openapi/paths/payment-methods.yaml

```
@@ -138,6 +165,6 @@ get:
                 type: boolean
             type: object
       description: ''
-  summary: List payment methods
+  summary: Get payment method
```

```suggestion
  summary: List payment methods
```
The request returns an array of payment methods (although, in the example, the array contains only a single payment method)